python/chronos/src/bigdl/chronos/data/experimental/xshards_tsdataset.py

```diff
@@ -364,6 +364,33 @@ def _inverse_transform(df, id_col, scaler, feature_col, target_col):
                                                   self.target_col)
         return self
 
+    def unscale_xshards(self, data):
+        '''
```

Comment:
add a parameter "key" here, default to None, in that case:
1. priority is "prediction" > "y"
2. if no "prediction" or "y" return an error and require our users to input a key